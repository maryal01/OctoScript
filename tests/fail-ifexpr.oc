int x = if (true) ? 1 : "two";