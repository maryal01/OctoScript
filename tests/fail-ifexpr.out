Fatal error: exception Failure("illegal if expression boolean int  string in if true 1 else "two"")